ci/checks/style.sh

```diff
@@ -38,10 +38,13 @@ source ci/common/prep-init-env.sh ${WORKSPACE}
 logger "Run Copyright header check..."
 ./ci/checks/check_copyright.py
 
+python -m pip install flake8
 # Run style check
 logger "Run Python formatting check..."
-python -m pip install flake8
 flake8 pyclaragenomics/
+# Run Cython style check
+logger "Run Cython formatting check..."
+flake8 pyclaragenomics/ --config pyclaragenomics/.flake8.cython
```

Comment:
What do you think of putting the two flake8 calls into a single script and calling that for python formatting? It would also make it easier for users in that the same command will cover both cython and python files